.github/workflows/pr.yml

```diff
@@ -143,6 +143,13 @@ jobs:
           sonar_utility: sonar-scanner
           sonar_commands: '("-Dsonar.projectKey=${{env.SONAR_PROJECT_KEY}} -Dsonar.host.url=${{env.SONAR_HOST_URL}} -Dsonar.login=${{env.SONAR_LOGIN}} -Dsonar.pullrequest.key=${{github.event.number}} -Dsonar.pullrequest.branch=${{github.event.pull_request.head.ref}} -Dsonar.pullrequest.base=${{github.event.pull_request.base.ref}}")'
 
+      - name: FrogBot
+        uses: lumada-common-services/gh-composite-actions@stable
+        env:
+          JF_URL: https://hitachi.jfrog.io/
+          JF_ACCESS_TOKEN: ${{ secrets.JF_ACCESS_TOKEN }}
+          JF_GIT_TOKEN: ${{ github.token }}
```

Comment:
Why do we need `JF_GIT_TOKEN` again? I don't remember... is it to allow Jfrog to comment on the PR maybe?!

Reply:
There is no specific mention from FrogBot regarding why this is required, but according to their [documentation,](https://github.com/jfrog/frogbot/blob/821a26f50338a319d238f9bbc054f6b7a66336ad/.github/workflows/frogbot-scan-pull-request.yml#L26) it is mandatory.